src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -126,7 +127,7 @@ def __getattr__(self, name: str):
             return _CaseVariable(self._variables, name + "/")
 
 
-class Case:
+class CaseFile:
```

Comment:
@hpohekar: Here @prmukherj is fixing a bug in this class and also changing the name. @prmukerj: is this about ready to get merged? Any genuine CI blocking it?

Reply:
@seanpearsonuk, this is ready from my end now. I was waiting to check if @tejalprabhu01 had any feedback on this.
Thank you.